Rivet/Functions/Operations/Add-Schema.ps1

```diff
@@ -24,10 +31,23 @@ function Add-Schema
         [Alias('Authorization')]
         [string]
         # The owner of the schema.
-        $Owner
+        $Owner,
+
+        [string]

```

Comment:
Style: only use all lower-case type names if it is a type accelerator. So `[String]` since that's the actual type name, but `[hashtable]` since that type's name is actually `[Collections.Hashtable]`.

Style: parameter type on the same line as parameter name, separated by a space.

Style: parameter comment should be above the parameter name, type, aliases, and other attributes.